recipe/meta.yaml

```diff
@@ -26,8 +26,9 @@ requirements:
 
 test:
   commands:
-    - test -f ${PREFIX}/include/nlohmann/json-schema.hpp  # [unix]
+    - test -f ${PREFIX}/include/nlohmann/json-schema.hpp
     - test -f ${PREFIX}/lib/libnlohmann_json_schema_validator.so  # [unix]
+    - test -f ${PREFIX}/lib/libnlohmann_json_schema_validator.dll  # [win]
```

Comment:
might also be good to check %PREFIX%\bin\nlohmann_json_schema_validator.bin` (I think).

Also ${PREFIX} isn't gonna work on win